test/package/packageVersionCreate.test.ts

```diff
@@ -593,6 +593,61 @@ describe('Package Version Create', () => {
     expect(package2DescriptorJson).to.have.string('buildOrgLanguage');
   });
 
```

Comment:
In order to get this test working I had to modify node_modules/@salesforoce/core/lib/sfProject.d.ts and add  to PackageDir:
   packageMetadataAccess?:  {
       permissionSets: string | string[];
       permissionSetLicenses: string | string[];
    };

Is there a corresponding change that needs to be submitted for that PackageDir definition?

Reply:
Yes, we just need to update this library to use the version of `@salesforce/core` (aka sfdx-core) that has the schema and type definition updates.

Reply:
The sfdx-core change has been merged: https://github.com/forcedotcom/sfdx-core/pull/1038